concordium-node/src/configuration.rs

```diff
@@ -458,88 +599,121 @@ pub struct CommonConfig {
         env = "CONCORDIUM_NODE_DATA_DIR"
     )]
     pub(crate) data_dir: PathBuf,
-    #[structopt(long = "no-log-timestamp", help = "Do not output timestamp in log output")]
+    #[structopt(
+        long = "no-log-timestamp",
+        help = "Do not output timestamp in log output",
+        env = "CONCORDIUM_NODE_NO_LOG_TIMESTAMP"
+    )]
     pub no_log_timestamp: bool,
     #[structopt(
         long = "minimum-peers-bucket",
         help = "Minimum peers to keep in each bucket always",
-        default_value = "100"
+        default_value = "100",
+        env = "CONCORDIUM_NODE_MINIMUM_PEERS_BUCKET"
     )]
     pub min_peers_bucket: usize,
-    #[structopt(long = "print-config", help = "Print out config struct")]
+    #[structopt(
+        long = "print-config",
+        help = "Print out config struct",
+        env = "CONCORDIUM_NODE_PRINT_CONFIG"
+    )]
     pub print_config: bool,
     #[structopt(
         long = "bucket-cleanup-interval",
         help = "Try to timeout entries in the buckets every set interval (in ms)",
-        default_value = "600000"
+        default_value = "600000",
+        env = "CONCORDIUM_NODE_BUCKET_CLEANUP_INTERVAL"
     )]
     pub bucket_cleanup_interval: u64,
 }
 
 // Client's parameters.
 #[derive(StructOpt, Debug)]
 pub struct CliConfig {
-    #[structopt(long = "no-network", help = "Disable network")]
+    #[structopt(
+        long = "no-network",
+        help = "Disable network",
+        env = "CONCORDIUM_NODE_CLIENT_NO_NETWORK"
+    )]
     pub no_network: bool,
     #[structopt(
         long = "poll-interval",
         help = "The polling interval in milliseconds",
-        default_value = "100"
+        default_value = "100",
+        env = "CONCORDIUM_NODE_CLIENT_POLL_INTERVAL"
     )]
     pub poll_interval: u64,
     #[structopt(flatten)]
     pub baker: BakerConfig,
     #[structopt(flatten)]
     pub rpc: RpcCliConfig,
     #[cfg(feature = "staging_net")]
-    #[structopt(long = "staging-net-token", help = "Staging network client token")]
+    #[structopt(
+        long = "staging-net-token",
+        help = "Staging network client token",
+        env = "CONCORDIUM_NODE_CLIENT_STAGING_NET_TOKEN",
+        hide_env_values = true
```

Comment:
Since there is no default value, what does this flag add?

Reply:
It hides the environment variable (if set). 

When invoking the "--help" on the node, then the environment variable would otherwise be printed.